### DISCUSSION TITLE
blog/post/typescript-challenges-1

### DISCUSSION THREAD

**Discussion by giscus** [The-Zagy/zagy-blog#15] [General]
# blog/post/typescript-challenges-1

Distributive conditional types in typescript

https://www.zagy.tech/blog/post/typescript-challenges-1

**Comment by kisk23** [comment]
Keep going 💪

**Comment by omar-ezzat** [comment]
Awesome ❤️❤️❤️

**Comment by Mohamed-H-Essa** [comment]
Interesting ❤️

**Comment by nagy-nabil** [comment]
My wizard going higher ❤️
